devlab/tests/testcases/test_verify_dst_del_tenant_resources.py

```diff
@@ -183,8 +183,8 @@ def test_tenants_volumes_on_dst(self):
                     {tenant_name: tenant_undeleted_volumes})
 
         if undeleted_volumes:
-            msg = 'Tenant\'s cinder volumes with ids {0} exist on destination,'\
-                  ' but should be deleted!'
+            msg = ('Tenant\'s cinder volumes with ids {0} exist on destination'
```

Comment:
Replace the single quote with double quotes ( ' -> " ) to avoid the extra backslash.